internal/controllers/submariner/submariner_controller.go

```diff
@@ -123,7 +123,6 @@ func NewReconciler(config *Config) *Reconciler {
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-
```

Comment:
This change doesn’t match the commit message.

Reply:
It actually fixes the error message indicated by the commit message. Without the blank line, it thinks L127 is the start of the method comment.